MSAL/src/native_auth/public/MSALNativeAuthPublicClientApplication.swift

```diff
@@ -116,57 +116,24 @@ public final class MSALNativeAuthPublicClientApplication: MSALPublicClientApplic
     /// Sign up a user with a given username and password.
```

Comment:
Is there a better way to describe this? As in "Sign up a user with a given username using OTP or Password"? Or just "Sign up a user with a given username"|

Reply:
I'd like to mention also the word "password" in this comment.  About your suggestion:
"Sign up a user with a given username using OTP or Password", with that "or" it seems that the OTP code can be substituted by the password, but an OTP is always required during sign up right now.
"Sign up a user with a given username", we should mention that a user can specifies a password.

I'd prefer to keep the description as it is, and specifies in the parameter list that the password is optional

Reply:
Ok, we can keep as is